### PR TITLE
Add sample recipes to customize openssl package

### PR DESCRIPTION
**Pull request by Yoshihiro-Okada-ML**
# Purpose of pull request

This commit adds sample recipes to customize openssl package.

### CODE REVIEW

**Comment by hiraku-wfs** [comment]
@Yoshihiro-Okada-ML Could you change filenames of *.bb to *.bb.sample?

**Comment by masami256** [comment]
@Yoshihiro-Okada-ML  I think it would be nice to simply describe what openssl-apt and openssl-http are in the README.md.

**Comment by Yoshihiro-Okada-ML** [comment]
Add short description for sample recipes.

File: README.md [hunk 1]
```diff
@@ -168,3 +168,14 @@ EMLinux currently supports following machines.
 - qemu-arm64
 - generic-x86-64
 - raspberrypi3bplus-64
+
+## Sample Recipe
+
+There are some sample recipes under doc/samples/recipes-sample/ directory. Currently you can see following sample recipes.
+
+- openssl-apt
```
Comment (hiraku-wfs) [inline README.md]: Please add `:` like `openssl-apt:`

> - openssl-apt This recipe downloads openssl source package using apt://. This recipe shows how to change build option and how to add new patch for source code.

> - openssl-apt: This recipe downloads openssl source package using apt://. This recipe shows how to change build option and how to add new patch for source code.

Comment (Yoshihiro-Okada-ML) [reply README.md]: add white space instead of ':'  I think this style is better.

Comment (hiraku-wfs) [reply README.md]: Thanks!

**Comment by hiraku-wfs** [comment]
Thanks!

**Pull request merged**